roles/dante/tasks/syslog.yml

```
@@ -32,9 +32,6 @@
           {{ commentout }}compress
           {{ commentout }}delaycompress
           sharedscripts
```

I think that using the "new" `rsyslog-rotate` command is a better workaround but let me know if it isn't
```suggestion
          sharedscripts
          postrotate
              /usr/lib/rsyslog/rsyslog-rotate
          endscript
```